src/main/java/com/esri/samples/ogc/browse_wfs_layers/BrowseWfsLayersSample.java

```diff
@@ -0,0 +1,243 @@
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.ogc.wfs.OgcAxisOrder;
+import com.esri.arcgisruntime.ogc.wfs.WfsFeatureTable;
+import com.esri.arcgisruntime.ogc.wfs.WfsLayerInfo;
+import com.esri.arcgisruntime.ogc.wfs.WfsService;
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+import com.esri.arcgisruntime.symbology.SimpleRenderer;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.control.ProgressIndicator;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class BrowseWfsLayersSample extends Application {
+
+  private MapView mapView;
+  private CheckBox checkBox;
+  private ArcGISMap map;
+  private ProgressIndicator progressIndicator;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    // create stack pane and JavaFX app scene
+    StackPane stackPane = new StackPane();
+    Scene scene = new Scene(stackPane);
+    scene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+    // set title, size, and add JavaFX scene to stage
+    stage.setTitle("Browse WFS Layers");
+    stage.setWidth(800);
+    stage.setHeight(700);
+    stage.setScene(scene);
+    stage.show();
+
+    // create a control panel
+    VBox controlsVBox = new VBox(6);
+    controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.3)"), CornerRadii.EMPTY, Insets.EMPTY)));
+    controlsVBox.setPadding(new Insets(10.0));
+    controlsVBox.setMaxSize(170, 200);
+    controlsVBox.getStyleClass().add("panel-region");
+
+    // create a check box to swap coordinate order
+    checkBox = new CheckBox("Swap coordinate order");
+    checkBox.setMaxWidth(Double.MAX_VALUE);
+    checkBox.setWrapText(true);
+
+    // create a label to display when checkbox is active: the WFSLayerInfos in this sample do not require coordinate swapping to display
+    Label label = new Label("Data in this sample doesn't require coord swapping to display. Uncheck to display data.");
+    label.setStyle("-fx-font-size:14px; -fx-text-fill: white;");
+    label.setVisible(false);
+
+    checkBox.setOnAction(event -> {
```

Comment:
you can omit the braces `{` for single line lamdas. Your IDE should hint this if your language level is set to 11.